### PR TITLE
Feature: user services

### PR DESCRIPTION
**Pull request by maasencioh**
Connects with:
  - [x] user_ms
  - [x] auth_ms
  - [x] notification_ms (depends of https://github.com/UNArqui17i-B/Notification/issues/6)

### CODE REVIEW

File: src/services/user.js [hunk 1]
```diff
@@ -1,3 +1,92 @@
...
+    const router = express.Router();
+
+    // Create a user
+    router.post('/', function (req, res) {
+        const user = req.body;
+        request({method: 'POST', url: userMicroService, json: user})
+            .then(() => request({
+                method: 'POST',
+                url: `${notificationMicroService}/confirmation`,
+                json: {
```
Comment (maasencioh) [inline src/services/user.js:22]: @NeillGiraldo the JSON is like this?

Comment (NeillGiraldo) [reply src/services/user.js:22]: Yes, that is perfect

File: src/serviceRegistry.js [hunk 2]
```diff
@@ -1,30 +1,30 @@
 'use strict';
 
 const users = {
-    user_ms: {
+    userMicroService: {
```
Comment (NeillGiraldo) [inline src/serviceRegistry.js:4]: Why the change of names?

Comment (maasencioh) [reply src/serviceRegistry.js:4]: my style checker only receives camellCase, but this is only in the code, not in the deployment

**Review by NeillGiraldo — APPROVED** [review]
Approved
